### PR TITLE
ability to drop metrics before prometheus

### PR DESCRIPTION
**Pull request by casey-green**
Ability to drop metrics before getting sent to prometheus

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -40,6 +40,8 @@ telegraf_influxdb_password:
 telegraf_influxdb_user_agent:
 telegraf_influxdb_udp_payload:
 
+telegraf_prometheus_metrics_filtered: []
```
Comment (mrwiggles) [inline defaults/main.yml]: Should we simply say `telegraf_metrics_filtered`?

Comment (casey-green) [reply defaults/main.yml]: and put `namedrop` in `output.influxdb` as well?

Comment (mrwiggles) [reply defaults/main.yml]: I think so, yes.

Comment (casey-green) [reply defaults/main.yml]: 👍

**Review by mrwiggles — APPROVED** [review]

**Pull request merged**